src/uiprotect/api.py

```diff
@@ -1451,25 +1451,152 @@ async def _stream_response(
             if progress_callback is not None:
                 await progress_callback(step, current, total)
 
-    async def get_camera_video(
+    async def download_camera_video(
+        self,
+        camera_id: str,
+        filename: str,
+        output_file: Optional[Path] = None,
```

Comment:
Please use pep 0563 style

pyupgrade can likely convert these for you

https://peps.python.org/pep-0563/

Reply:
https://github.com/asottile/pyupgrade